Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -1173,10 +1181,6 @@ def dlconfig_changed_callback(self, section, name, new_value, old_value):
     def get_share_mode(self):
         return self.handle.status().share_mode
 
-    @waitForHandleAndSynchronize(True)
-    def set_share_mode(self, share_mode):
```

Comment:
Note that @ardhipoetra has to change several calls in his credit mining code if we do this.